src/App.scss

```diff
@@ -6,3 +6,43 @@ html {
   font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
   font-size: 24px;
 }
+
+body {
+  margin: 0;
+}
```

Comment:
These parameters probably should be in normilize file

Reply:
Thank you, done)